access_control.py

```diff
@@ -0,0 +1,225 @@
+"""
+This script will process the `grant_access/` and `revoke_access/` requests.
+
+Main logic lives in conda-smithy. This is just a wrapper for admin-requests infra.
+"""
+import os
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+import requests
+from ruamel.yaml import YAML
+
+GH_ORG = os.environ.get("GH_ORG", "conda-forge")
+
+CIRUN_FILENAME_RESOURCE_MAPPING = {
+    "cirun-gpu-runner": {
+        "resource": "cirun-gpu-runner",
+    },
+    "cirun-gpu-runner-pr": {
+        "resource": "cirun-gpu-runner",
+        "policy_args": ["pull_request"]
+    }
+}
+
+
+ACCESS_YAML_FILENAME = ".access_control.yml"
+
+
+def _get_access_control_files(path):
```

Comment:
Let's add simple docstrings and minimal type hints to all functions. This is the "get the input files" function, right?